app/classes/api2/species_list_api.rb

```diff
@@ -108,12 +108,12 @@ def build_setter(params)
     private
 
     def validate_set_location!(params)
-      name = params[:place_name].to_s || return
+      name = params[:place_name].to_s
```

Comment:
This seems risky.

Reply:
`to_s` should always be truthy, so `return` should never get evaluated.

Reply:
Got it.